docker/sui-graphql-rpc/Dockerfile

```diff
@@ -19,15 +20,11 @@ RUN cargo build --profile ${PROFILE} --bin sui-graphql-rpc
 
 # Production Image
 FROM debian:bullseye-slim AS runtime
-# Use jemalloc as memory allocator
-RUN apt-get update && apt-get install -y libjemalloc-dev ca-certificates
-ENV LD_PRELOAD /usr/lib/x86_64-linux-gnu/libjemalloc.so
-ARG PROFILE=release
 WORKDIR "$WORKDIR/sui"
 # Both bench and release profiles copy from release dir
+RUN mkdir -p /opt/sui/bin
 COPY --from=builder /sui/target/release/sui-graphql-rpc /opt/sui/bin
-# Support legacy usages of /usr/local/bin/sui-graphql-rpc
-COPY --from=builder /sui/target/release/sui-graphql-rpc /usr/local/bin
```

Comment:
Any reason why we dont need this anymore?

Reply:
i think that was copied from `sui-node`, we probably still need it there but as this is a new image/service we can just default to whatever we prefer and not need two locations